apps/backend/scripts/clickhouse-migrations.ts

```diff
@@ -0,0 +1,47 @@
+import { clickhouseAdminClient } from "@/lib/clickhouse";
+import { getEnvVariable } from "@stackframe/stack-shared/dist/utils/env";
+
+export async function runClickhouseMigrations() {
+  console.log("[Clickhouse] Running Clickhouse migrations...");
+  const client = clickhouseAdminClient;
+  const clickhouseExternalPassword = getEnvVariable("STACK_CLICKHOUSE_EXTERNAL_PASSWORD");
+  await client.exec({
+    query: "CREATE USER IF NOT EXISTS limited_user IDENTIFIED WITH plaintext_password BY {clickhouseExternalPassword:String}",
+    query_params: { clickhouseExternalPassword },
+  });
+  // todo: create migration files
+  await client.exec({ query: EVENTS_TABLE_BASE_SQL });
+  await client.exec({
+    query: "CREATE ROW POLICY IF NOT EXISTS events_project_isolation ON events FOR SELECT USING project_id = getSetting('SQL_project_id') AND branch_id = getSetting('SQL_branch_id') TO limited_user",
+  });
+  const queries = [
+    "GRANT SELECT ON analytics.events TO limited_user;",
+    "REVOKE ALL ON system.* FROM limited_user;",
+    "REVOKE CREATE, ALTER, DROP, INSERT ON *.* FROM limited_user;"
+  ];
```

Comment:
could we do

```suggestion
  const queries = [
    "REVOKE ALL PRIVILEGES ON *.* FROM limited_user;",
    "REVOKE ALL FROM limited_user;"
    "GRANT SELECT ON analytics.events TO limited_user;",
  ];
```

just to make sure we revoke everything?